src/python/grpcio/grpc/_channel.py

```diff
@@ -926,6 +926,11 @@ def __init__(self, target, options, credentials):
         self._call_state = _ChannelCallState(self._channel)
         self._connectivity_state = _ChannelConnectivityState(self._channel)
 
+        # Temporary work around UNAVAILABLE issues
```

Comment:
This should be a `TODO` (possibly a `TODO(https://github.com/grpc/grpc/issues/11043)`?).